### PR TITLE
Chakra v1

### PR DESCRIPTION
**Pull request by joelwass**

### CODE REVIEW

File: components/common/backButton.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <Link onClick={() => router.push(destination)}>
+      <Flex flexDirection='row' position='absolute' zIndex={1000} top='10rem' left='5rem' padding='1rem' cursor='pointer'>
+        <ArrowBackIcon w='1.5rem' h='1.5rem' />
+        <Text paddingLeft='1rem'>Back</Text>
+      </Flex>
+    </Link>
+  )
+}
+
+export default BackButton
```
Comment (stripedpajamas) [inline components/common/backButton.js:22]: yasss, saw this and loved it

File: components/common/banner.js [hunk 2]
```diff
@@ -20,12 +20,14 @@ const Banner = ({ icon, children, onCloseClick }) => {
       borderColor='ocean'
       borderBottom='1px'
     >
-      <Icon
-        name={icon}
-        size={{ base: '2rem', md: '3rem' }}
-        marginRight={{ base: 0, md: '1.5rem' }}
-        marginBottom={{ base: '1.5rem', md: 0 }}
-      />
+      {icon === 'givingHand' && (
```
Comment (stripedpajamas) [inline components/common/banner.js:23]: what's going on here ? looks like a weird way to accomplish this

File: public/theme.js [hunk 3]
```diff
@@ -1,21 +1,22 @@
...
     "DejaVu Sans Condensed", "Liberation Sans", "Nimbus Sans L", Tahoma, Geneva,
     Arial, sans-serif`
 
-const breakpoints = ['30em', '48em', '62em', '80em']
-// aliases
-breakpoints.sm = breakpoints[0]
-breakpoints.md = breakpoints[1]
-breakpoints.lg = breakpoints[2]
-breakpoints.xl = breakpoints[3]
+const breakpoints = createBreakpoints({
```
Comment (stripedpajamas) [inline public/theme.js:11]: ahhh yeah i think this is where i gave up with maintainer portal.

**Review by stripedpajamas — APPROVED** [review]
awezome

File: components/common/banner.js (hunk 2 above)
Comment (joelwass) [reply components/common/banner.js:23]: Ya help me find a better way 😃

**Pull request merged**